backend/src/matches/matches.service.ts

```diff
@@ -36,7 +36,7 @@ export class MatchesService {
     const findOptionsWhere: FindOptionsWhere<Match> = {
       home: query?.homeId ? { id: +query.homeId } : {},
       away: query?.awayId ? { id: +query.awayId } : {},
-      status: query?.role,
+      status: query?.status,
```

Comment:
That''s good enough to fix the issue, but since I took a look at the queryDto I have two questions:
- Is it normal/ok for the MatchDto to be actually be called Match rather than MatchDto ? After all, the Match entity is already called Match, as it should..
- Is there a reason why we need to redefine queryDto from the ground up rather than using pickType ? I feel like it could simply be : 
```
export class QueryMatchDto extends PickType(MatchDto, [
'homeId',
'awayId',
'status'
])
{};
```
But maybe there's something I don't see

Reply:
(In case you agree with this feel free to still merge your PR and I'll do a quick one right after).

Reply:
I think it is outside of the scope of the PR but you make valid points, let us fix this in another PR.